### PR TITLE
chore: set release date in changelog to 2 days from branch cut @W-10785024@

### PR DESCRIPTION
**Pull request by jag-sfdc**
### What does this PR do?
Reduces work for the doc team by setting the release date to 2 days from branch cut + minor formatting

### What issues does this PR fix or reference?
@W-10785024@

### Functionality Before
N/A

### Functionality After
No change in functionality

### CODE REVIEW

File: scripts/change-log-generator-utils.js [hunk 1]
```diff
@@ -274,27 +274,17 @@ function generateKey(packageName, type, packagesToIgnore) {
...
+  let releaseDate = new Date();
+  releaseDate.setDate(releaseDate.getDate() + 2);
+  return new Intl.DateTimeFormat('en-US', {
+      month: 'long',
+      day: 'numeric',
+      year: 'numeric'
+    }).format(releaseDate);
 }
 
-function getArgumentValue(arg) {
```
Comment (jag-sfdc) [inline scripts/change-log-generator-utils.js:287]: This is a function used for the old changelog generation. Not required anymore.

File: scripts/change-log-generator-utils.js [hunk 2]
```diff
@@ -274,27 +274,17 @@ function generateKey(packageName, type, packagesToIgnore) {
...
-      }).format(new Date());
+  // We want to ideally release two days from the day the release branch is cut
+  // (typically branch cut happens Monday and release on Wednesday)
+  let releaseDate = new Date();
+  releaseDate.setDate(releaseDate.getDate() + 2);
+  return new Intl.DateTimeFormat('en-US', {
+      month: 'long',
+      day: 'numeric',
+      year: 'numeric'
+    }).format(releaseDate);
```
Comment (gbockus-sf) [inline scripts/change-log-generator-utils.js:285]: should we be removing the ability to override with the arg `RELEASE_DATE_ARG`?

Comment (randi274) [reply scripts/change-log-generator-utils.js:285]: If we're ok with removing it, we can also remove the unused const RELEASE_DATE_ARG.

Comment (jag-sfdc) [reply scripts/change-log-generator-utils.js:285]: Yes, this arg was passed in during changelog generation manually. We don't need this anymore.

**Review by randi274 — APPROVED** [review]

**Pull request merged**